scripts/202306xx_populate_paymenttype_default/add_fixed_payment_type_default.js

```diff
@@ -4,31 +4,87 @@
 Add PaymentType.Fixed to all forms with existing payments_field object
 */
 
-// Count number of forms that contains payments_field property
-db.getCollection('forms')
-  .find({ payments_field: { $exists: true } })
-  .count()
+// Batch 1 2022
+const queriesModifiedIn2022 = {
+  payments_field: { $exists: true },
+  lastModified: { $gte: new ISODate('2022-01-01T00:00:00.000+0000') },
+  lastModified: { $lt: new ISODate('2023-01-01T00:00:00.000+0000') },
+}
 
-// Count number of forms that contains payments_field.payment_type property.
-// Expected to be 0
-db.getCollection('forms')
-  .find({ "payments_field.payment_type": { $exists: true } })
-  .count()
+const queriesModifiedIn2023Q1 = {
+  lastModified: { $gte: new ISODate('2023-01-01T00:00:00.000+0000') },
+  lastModified: { $lt: new ISODate('2023-04-01T00:00:00.000+0000') },
+}
 
-/*
-Create payments_field.payment_type from existing payments_fields
+const queriesModifiedIn2023AprMay = {
+  lastModified: { $gte: new ISODate('2023-04-01T00:00:00.000+0000') },
+  lastModified: { $lt: new ISODate('2023-06-01T00:00:00.000+0000') },
+}
+
+const queriesModifiedIn2023JunOnwards = {
+  lastModified: { $gte: new ISODate('2023-06-01T00:00:00.000+0000') },
+}
+
+function executePaymentTypeInsertionOnQuery(dateQuery, dryRun = true) {
```

Comment:
Usually on our shell we will 'step' into each of the query right, but are we able to do that when we wrap it in a function? 

Or is that why we have the `dryRun` break 😅

Reply:
Yeah, that's what I'm intending the `dryRun` to be. I was thinking that we have quite a few batched queries to run, and copy/pasting might be a point of risk / error.

Reply:
Oh I mean we use F6 right? at least for studio 3T

So like you copy paste the entire script then F6 all the way down 😅

<img width="914" alt="image" src="https://github.com/opengovsg/FormSG/assets/59867455/a7f3c22d-33c1-471f-8449-3c9741465a41">

But I think the `dryRun` method is fine too! As it separates the check and the execution which is the main focus anyway hahaha

Reply:
I... actually didn't know of this F6 on Studio 3T 🙈. I think it makes sense to use that.